org.eclipse.lemminx/pom.xml

```diff
@@ -11,7 +11,7 @@
 		<maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
 		<dev.build.timestamp>${maven.build.timestamp}</dev.build.timestamp>
 		<cbi.jarsigner.skip>true</cbi.jarsigner.skip>
-		<graalvm.version>20.3.0</graalvm.version>
+		<native.maven.plugin.version>0.9.9</native.maven.plugin.version>
```

Comment:
try with 0.9.10

Reply:
And the commit message should be updated as well since it mentions 0.9.9